multiqc/modules/verifybamid/verifybamid.py

```diff
@@ -0,0 +1,238 @@
+                                parsed_data[s_name][headers[i]] = v
+    
+        # nothing has been parsed return None
+        if len(parsed_data) == 0:
+            return None
+        # else return the dictionary
+        return parsed_data
+
+    def verifybamid_general_stats_table(self):
+        """ Take the percentage of contamination from all the parsed *.SELFSM files and add it to the basic stats table at the top of the report """
+
+        # create a dictionary to hold the columns to add to the general stats table
+        headers = OrderedDict()
+        # available columns are:
+        #SEQ_ID RG  CHIP_ID #SNPS   #READS  AVG_DP  FREEMIX FREELK1 FREELK0 FREE_RH FREE_RA CHIPMIX CHIPLK1 CHIPLK0 CHIP_RH CHIP_RA DPREF   RDPHET  RDPALT
+        #,see https://genome.sph.umich.edu/wiki/VerifyBamID#Interpreting_output_files
+
+        # add the CHIPMIX column. set the title and description
+        headers['CHIPMIX'] = {
+            'title': 'Contamination Prediction - sequence and array',
+            'description': 'VerifyBamID: CHIPMIX -   Sequence+array estimate of contamination (NA if the external genotype is unavailable) (0-1 scale)',
+            'format': '{:,.5f}' 
+        }
+
+        # add the FREEMIX column. set the title and description
+        headers['FREEMIX'] = {
+            'title': 'Contamination Prediction - sequence only',
+            'description': 'VerifyBamID: FREEMIX -   Sequence-only estimate of contamination (0-1 scale).',
+            'format': '{:,.5f}' 
+        }
+        
+        # pass the data dictionary and header dictionary to function to add to table.
+        self.general_stats_addcols(self.verifybamid_data, headers)
+
+        
+    def verifybamid_table(self):
+        """
+        Create a table with all the columns from verify BAM ID
+        """
+        # create empty dictionary
+        data = dict()
+        # for each smaple name and the corresponding dictionary
+        for s_name, d in self.verifybamid_data.items():
+            # create a entry in data dictionary where key is sample namd and value an empty dictionary
+            data[s_name]={}
+            # for each column for this sample
+            for column in d:
+                # add the column name to the sample dictionary and the associated value
+                data[s_name][column] = d[column]
+
+        # create an ordered dictionary to preserve the order of columns
+        headers = OrderedDict()
+        # add each column and the title and description (taken from verifyBAMID website)
+        headers['#SEQ_ID']={
+            'title': 'Sample Name',
+            'description': 'Sample ID of the sequenced sample. Obtained from @RG header / SM tag in the BAM file',
+        }
+        headers['RG']={
+            'title': 'Read Group',
+            'description': 'ReadGroup ID of sequenced lane. For [outPrefix].selfSM and [outPrefix].bestSM, these values are "ALL',
```

Comment:
Oops! You've just highlighted a minor MultiQC bug for me, where I need to escape double quotes in descriptions (this is truncated after `are "`).

Anyway - is this always `ALL`? If so, I guess it's not very interesting and we can skip this column? (Just omit the header config for the dictionary key to skip it in the table).